### DISCUSSION TITLE
does vanjs support foreignObject?

### DISCUSSION THREAD

**Discussion by kangaroolab** [vanjs-org/van#381] [General]
I tried and it returns outerHTML as `<foreignobject></foreignobject>`. Should it be with capital "O" like `<foreignObject></foreignObject>`?

**Comment by sirenkovladd** [comment]
here is example from https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject
https://jsfiddle.net/Sirenko/bvt3wpLg/
![image](https://github.com/user-attachments/assets/f6a49e30-cf28-4ac1-8693-0b9b99f9f3ff)

**Comment by kangaroolab** [comment]
which version of vanjs you used? with my version, I got following:

```
onsole.log(foreignObject());
VM3727:1 <foreignobject>​</foreignobject>​
```

with your version, I got the right HTML tag with big "O". 

my version is https://cdn.jsdelivr.net/gh/vanjs-org/van/public/van-1.5.2.nomodule.min.js. is there a bug?

**Reply by sirenkovladd** [reply]
the same version, can you send a reproduction?

**Comment by kangaroolab** [comment]
[see here](https://flems.io/#0=N4IgzgpgNhDGAuEAmIBcIB0ALeBbKIANCAGYCWMYaA2qAHYCGuEamO+RIsA9nYn6wA8YWACcyAB3gACMKNgBeADohpOeBLCoA9NthI6GAFZgk0MgDdRGOhHjaA5lm0WGdEwFpuohy7faJAFcAIygyWD86DwBGDABWDAAmG25cbiRAmAxcMkMTFQA+QW0RcSkCpTphMUl4CrppaR46MBlgEm8IMgc6AHlgozh4AF9pBWlXQ3gGBzAAbkrG5rBuLKhuBwAKDtEunv7BhE2ASmOFqpKa8s5IGAQyXip0AGZUZ+iQYcJ6JhZ0YyoxGa-HgrE+3xAjGYrAwsDAgK4vBBYOGAF1hkA)

**Reply by sirenkovladd** [reply]
```js
const {foreignObject} = van.tags("http://www.w3.org/2000/svg");
console.log(foreignObject().outerHTML);
```

this code looks good

**Comment by kangaroolab** [comment]
thanks
